### ISSUE TITLE
Improvement Documentation

### ISSUE THREAD

**Issue by Drallas** [Kovah/LinkAce-Website#29]
Hi Kevin

I struggled a bit setting up LinkAce, finding the documentation not always up to date or consistent. 

If you are open to it I could help a bit to improve it, amd make it easier for other to get LinkAce up and running?
 
Got the Hugo site already running locally inside a devcontainer; so I'm ready to go!

**Comment by Kovah** [comment]
Of course, feel free to edit. I might be a little bit blind to some issues in the docs, the last time I have really set up the app was earlier last year. 😅

**Comment by Drallas** [comment]
I already did one page, for you to gauge if my style is ok! 😃

Have more ideas how to improve some things, but need some time to refine!
